packages/strapi-plugin-users-permissions/admin/src/translations/es.json

```diff
@@ -0,0 +1,175 @@
+  "HeaderNav.link.roles": "Roles y Permisos",
+
+  "HomePage.header.title": "Roles y Permisos",
+  "HomePage.header.description": "Defina los roles y permisos para sus usuarios.",
+
+  "InputSearch.placeholder": "Buscar un usuario",
+
+  "List.button.roles": "Añadir nuevo rol",
+  "List.button.providers": "Añadir nuevo proveedor",
+
+  "List.title.emailTemplates.singular": "{number} plantilla de email está disponible",
+  "List.title.emailTemplates.plural": "{number} plantillas de email disponibles",
+
+  "List.title.providers.disabled.singular": "{number} está desactivado",
+  "List.title.providers.disabled.plural": "{number} están desactivados",
+  "List.title.providers.enabled.singular": "{number} está habilitado y",
+  "List.title.providers.enabled.plural": "{number} proveedores están habilitados y",
+
+  "List.title.roles.singular": "{number} rol disponible",
+  "List.title.roles.plural": "{number} roles disponibles",
+
+  "notification.error.delete": "Se ha producido un error al intentar borrar el elemento",
+  "notification.error.fetch": "Se ha producido un error al intentar recuperar los datos",
+  "notification.error.fetchUser": "Se ha producido un error al intentar buscar usuarios",
+  "notification.info.emailSent": "El email ha sido enviado",
+  "notification.success.delete": "El elemento ha sido borrado",
+  "notification.success.submit": "Los ajustes se han actualizado",
+
+  "plugin.description.short": "Proteja su API con un proceso de autenticación completo basado en JWT",
+  "plugin.description.long": "Proteja su API con un proceso de autenticación completo basado en JWT. Este plugin viene también con una estrategia ACL que le permite administrar los permisos entre los grupos de usuarios.",
+
+  "Plugin.permissions.application.description": "Defina todas las acciones permitidas de su proyecto.",
+  "Plugin.permissions.plugins.description": "Defina todas las acciones permitidas para el plugin {name}.",
+
+  "Plugins.header.title": "Permisiones",
+  "Plugins.header.description": "Sólo las acciones vinculadas a una ruta se enumeran a continuación.",
+
+  "Policies.InputSelect.empty": "Ninguno",
+  "Policies.InputSelect.label": "Permita que se realice esta acción para:",
+  "Policies.header.hint": "Seleccione las acciones de la aplicación o las acciones del plugin y haga clic en el icono del engranaje para ver la ruta vinculada",
+  "Policies.header.title": "Ajustes avanzados",
+
+  "Email.template.validation_email": "Validación de la dirección de email",
+  "Email.template.reset_password": "Restablecer contraseña",
+  "Email.template.success_register": "Inscripción exitosa",
+
+  "Auth.advanced.allow_register": "",
+
+  "PopUpForm.button.cancel": "Cancelar",
+  "PopUpForm.button.save": "Guardar",
+  "PopUpForm.header.add.providers": "Añadir nuevo proveedor",
+  "PopUpForm.header.edit.email-templates": "Editar Plantillas de Email",
+  "PopUpForm.header.edit.providers": "Editar {provider} Proveedor",
```

Comment:
🤔 Again, without see where is used, I think could be better "Editar Proveedor {provider}"